lib/sort_by_length.rb

```diff
@@ -1,7 +1,57 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n+n^2)
+# Space complexity: O(n)
 def sort_by_length(my_sentence)
```

Comment:
Good implementation of Bubblesort although your time complexity is O(n^2) as we drop the smaller term.

Also you could have simply used `split` to break up the string into individual words